loopd/swapclient_server.go

```diff
@@ -83,8 +82,9 @@ func (s *swapClientServer) LoopOut(ctx context.Context,
 		}
 	}
 
-	// Check that the label is valid.
-	if err := labels.Validate(in.Label); err != nil {
+	sweepConfTarget, err := validateLoopOutRequest(in.SweepConfTarget,
```

Comment:
See style guide for wrapping rule: https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#additional-style-constraints-on-top-of-gofmt